### PR TITLE
Increase baremetalds-packet-check number of seconds for Assisted

### PR DESCRIPTION
**Pull request by omertuc**
For assisted-installer related jobs, due to the hub-spoke nature, we
usually have to run more than one OpenShift installation as part of the
tests. This means packet machines in assisted jobs are needed for longer
and so the threshold of when a server is considered "leaked" should be
increased from 4 to 5 hours.

This commit makes it so that the baremetalds-packet-check step is
configurable so the of time before a server is considered a leak can be
set differently for each packet check job. The assisted jobs will now
use 5 hours for that step.

Also deleted assisted-packet-check and sno-packet-check from the
release-ocm-2.5 branch config because it's enough to just have it
on the master branch config, those machines are shared by all branches
anyway so cleaning leaks on one branch cleans for the others as well.

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -86,6 +86,8 @@ tests:
     cluster_profile: packet-sno
     test:
     - ref: baremetalds-packet-check
+    env:
+      LEAK_THRESHOLD_SECONDS: 5:0:0
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:90]: it's actually more important to apply that for ``assisted-packet-check`` than for ``sno-packet-check``
I'd say we should apply it for both for consistency

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml [hunk 2]
```diff
@@ -65,6 +65,8 @@ tests:
   cron: 0 */2 * * *
   steps:
     cluster_profile: packet-assisted
+    env:
+      LEAK_THRESHOLD_SECONDS: 5:0:0
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml:69]: I'm actually not sure why we have this job for 2.5 branch. I think it's only needed once per cluster-profile

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml (hunk 2 above)
Comment (omertuc) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml:69]: True, thought about it. But it's just gonna get cloned again next time a bot / person creates the next release config file

Comment (omertuc) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml]: Deleted them

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml (hunk 1 above)
Comment (omertuc) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml]: ah oops, meant to put it on assisted-packet-check

Comment (omertuc) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml]: sno packet is just for the live-iso stuff, it doesn't need such a long time

Comment (osherdp) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml]: right

**Review by osherdp — APPROVED** [review]

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml (hunk 2 above)
Comment (osherdp) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-release-ocm-2.5.yaml]: Right. Last time we pretty much copied everything, lots of it was redundant when considering the purpose of this branch

**Comment by osherdp** [comment]
I guess you need ``make jobs``

**Review by osherdp — APPROVED** [review]

**Pull request merged**